[Pages] Document service bindings (#6561)
cloudflare/cloudflare-docs: content/pages/platform/functions/_index.md

}
```

> ### Service binding

Author: ```suggestion
### Service binding
```

> ### Service binding

Author: We use a singular for all the others too

> ### Service bindings

Author: ```suggestion
### Service binding
```

> Service bindings allow for you to call a Worker from within your Function with zero latency. You can add a service binding to your project by going to **Account Home** > **Pages** > **your Pages project** > **Settings** > **Functions** > **Service binding**. Select a Service from the list of your existing Workers and then select the environment. You must repeat this step for both the **Production** and **Preview** environments.

Reviewer: ```suggestion
Service bindings allow for you to call a Worker from within your Function with zero latency. To add a service binding to your project: 

1. Log into the [Cloudflare dashboard](https://dash.cloudflare.com).
2. In **Account Home**, select **Pages** > **your Pages project** > **Settings** > **Functions** > **Service binding**. 
3. Select a service from the list of your existing Workers and then select the environment. You must repeat this step for both the **Production** and **Preview** environments.
```

> ![Editing a Service Binding and adding a Variable name](/pages/platform/media/)

Reviewer: ```suggestion
![Follow the instructions above to edit a Service Binding and add a Variable name](/pages/platform/media/)
```

> ### Using Service bindings locally

Reviewer: ```suggestion
### Use Service bindings locally
```

}
```

### Service binding

Service bindings allow for you to call a Worker from within your Function with zero latency. To add a service binding to your project: 

1. Log into the [Cloudflare dashboard](https://dash.cloudflare.com).
2. In **Account Home**, select **Pages** > **your Pages project** > **Settings** > **Functions** > **Service bindings**. 
3. Select a service from the list of your existing Workers and then select the environment. You must repeat this step for both the **Production** and **Preview** environments.

> 
> ![Follow the instructions above to edit a Service Binding and add a Variable name](/pages/platform/media/)

Author: ```suggestion
```
